database/models.ts

```diff
@@ -24,16 +24,16 @@ export interface AppUser {
   Streak: number;
   CurrencyAmount: number;
   LastEntry: number;
```

Comment:
What does this refer to? Is this a date?

Relatedly, I'm seeing some dates that are strings and some that are numbers.